terraform/modules/google-cloud/apps/elixir/main.tf

```diff
@@ -153,8 +153,10 @@ resource "google_compute_instance_template" "application" {
   }
 
   network_interface {
-    subnetwork = var.vpc_subnetwork
-    stack_type = "IPV4_IPV6"
+    subnetwork  = var.vpc_subnetwork
+    nic_type    = "GVNIC"
+    queue_count = var.queue_count
```

Comment:
What does the queue count actually do? Unless we need it, should we leave it at default?

Reply:
It sets the number of maximum (and active by default) RX / TX packet queues. From what I've researched online, setting this equal to the number of CPU cores (up to a point) can improve performance because the kernel interrupt scheduler can then assign one set of RX / TX queue per core.

Reply:
It's strange that XDP requires this to be half of the maximum. I suspect that has something to do with perhaps XDP affecting the scheduler somehow.

Reply:
GCP enforces the max here to be equal to the number of vCPUs, so `2` is a good number since our modules all default to `e2-micro`.